sdk/storage/azure-storage-files-datalake/src/datalake_sas_builder.cpp

```diff
@@ -223,8 +228,8 @@ namespace Azure { namespace Storage { namespace Sas {
         + "\n" + userDelegationKey.SignedService + "\n" + userDelegationKey.SignedVersion + "\n"
         + PreauthorizedAgentObjectId + "\n" + AgentObjectId + "\n" + CorrelationId + "\n"
         + (IPRange.HasValue() ? IPRange.Value() : "") + "\n" + protocol + "\n" + SasVersion + "\n"
-        + resource + "\n" + "\n" + CacheControl + "\n" + ContentDisposition + "\n" + ContentEncoding
-        + "\n" + ContentLanguage + "\n" + ContentType;
```

Comment:
Silly question: I don't seem to find anything in this code which digitally signs `stringToSign`. Would it be better named as `stringToEncode`?

Similarly, the variable `signature` doesn't appear to be a digital signature.

Reply:
I think HMAC-256 can be considered a signing process.